web/src/compose_closed_ui.ts

```diff
@@ -137,14 +137,40 @@ export function update_buttons_for_private(): void {
     update_buttons(disable_reply);
 }
 
+function get_current_dm_recipient_ids_string(): string {
+    const selected_message = message_lists.current?.selected_message();
+    const recipient_id = selected_message?.display_recipient;
+
+    if (Array.isArray(recipient_id)) {
+        return recipient_id.map((recipient: {id: number}) => recipient.id).join(",");
+    }
```

Comment:
I think you can use an existing method to get recipient ids from display_recipient instead of introducing a new method. See other code where `user_can_send_direct_message` is used.